datafusion/core/src/execution/context.rs

```diff
@@ -351,88 +352,19 @@ impl SessionContext {
                 }
                 self.return_empty_dataframe()
             }
-            LogicalPlan::CreateExternalTable(cmd) => {
-                self.create_external_table(&cmd).await
-            }
-
-            LogicalPlan::CreateMemoryTable(CreateMemoryTable {
```

Comment:
I moved this code into separate methods as this particular dispatch method was getting quite large